src/components/dictionaryConcepts/components/ActionButtons.jsx

```diff
@@ -23,9 +22,6 @@ const ActionButtons = ({
           >
           Edit
           </Link>
-          <AddMapping
```

Comment:
Is this related related to remove the "show all button"?

Reply:
No. I can't remember making these changes. This must be a mistake.

Reply:
Can you go through all these changes and make sure you do not include any unneeded ones?

Reply:
I have taken care of it.